lib/ansible/plugins/test/regex.yml

```diff
@@ -22,14 +22,27 @@ DOCUMENTATION:
       type: boolean
       default: False
     match_type:
-      description: Decide which function to be used to do the matching.
+      description:
+        - Decide which function to be used to do the matching.
+        - The V(fullmatch) is added version Ansible Core 2.20.
```

Comment:
Although the`fullmatch` option was documented in 2.20, it has always been available, so I don't think it should be mentioned here.